apache/metrics.go

```diff
@@ -0,0 +1,62 @@
+/*
+http://www.apache.org/licenses/LICENSE-2.0.txt
+
+
+Copyright 2016 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apache
+
+import "github.com/intelsdi-x/snap-plugin-lib-go/v1/plugin"
+
+var apacheMetrics = [][]string{
+	[]string{"intel", "apache", "BusyWorkers"},
+	[]string{"intel", "apache", "BytesPerReq"},
+	[]string{"intel", "apache", "BytesPerSec"},
+	[]string{"intel", "apache", "ConnsAsyncClosing"},
+	[]string{"intel", "apache", "ConnsAsyncKeepAlive"},
+	[]string{"intel", "apache", "ConnsAsyncWriting"},
+	[]string{"intel", "apache", "ConnsTotal"},
+	[]string{"intel", "apache", "CPULoad"},
+	[]string{"intel", "apache", "IdleWorkers"},
+	[]string{"intel", "apache", "ReqPerSec"},
+	[]string{"intel", "apache", "Total_Accesses"},
+	[]string{"intel", "apache", "Total_kBytes"},
+	[]string{"intel", "apache", "Uptime"},
+	[]string{"intel", "apache", "workers", "Closing"},
+	[]string{"intel", "apache", "workers", "DNSLookup"},
+	[]string{"intel", "apache", "workers", "Finishing"},
+	[]string{"intel", "apache", "workers", "Idle_Cleanup"},
+	[]string{"intel", "apache", "workers", "Keepalive"},
+	[]string{"intel", "apache", "workers", "Logging"},
+	[]string{"intel", "apache", "workers", "Open"},
+	[]string{"intel", "apache", "workers", "Reading"},
+	[]string{"intel", "apache", "workers", "Sending"},
+	[]string{"intel", "apache", "workers", "Starting"},
+	[]string{"intel", "apache", "workers", "Waiting"},
+}
+
+// getApacheMetrics returns a slice of available plugin.Metric created
+// from the apacheMetrics array
+func getApacheMetrics(config plugin.Config) []plugin.Metric {
+	var mts []plugin.Metric
+	for _, k := range apacheMetrics {
+		mts = append(mts, plugin.Metric{
+			Namespace: plugin.NewNamespace(k...),
+			Config:    config,
```

Comment:
If it's defined in the global snap config does it need to be passed here? I'm fine with removing it, I just wanted to make sure it won't break it for people who are defining it there